tests/nosetests/pyanaconda_tests/module_storage_checker_test.py

```diff
@@ -39,3 +39,58 @@ def set_constraint_test(self):
         with patch.dict(storage_checker.constraints):
             self.interface.SetConstraint(STORAGE_MIN_RAM, get_variant(Int, 987))
             self.assertEqual(storage_checker.constraints[STORAGE_MIN_RAM], 987)
+
+
+class StorageCheckerVerificationTestCase(unittest.TestCase):
+
+    @patch("pyanaconda.storage.checker._", side_effect=lambda x: x)
+    def lvm_verification_test(self, _):
+        """Test the LVM destruction test."""
+        # VG that is destroyed correctly
+        action1 = Mock()
+        action1.is_destroy = True
+        action1.is_device = True
+        action1.device.type = "lvmvg"
+        action1.device.name = "VOLGROUP-A"
+
+        # something to ignore
+        action2 = Mock()
+        action2.is_destroy = False
+
+        # PV that belongs to VG from #1
+        action3 = Mock()
+        action3.is_destroy = True
+        action3.is_format = True
+        action3.orig_format.type = "lvmpv"
+        action3.device.disk.name = "PHYSDISK-1"
+        action3.orig_format.vg_name = "VOLGROUP-A"
+
+        # PV that belongs to a missing group
+        action4 = Mock()
+        action4.is_destroy = True
+        action4.is_format = True
+        action4.orig_format.type = "lvmpv"
+        action4.device.disk.name = "PHYSDISK-2"
+        action4.orig_format.vg_name = "VOLGROUP-B"
+
+        # PV that does not belong to any group
+        action5 = Mock()
+        action5.is_destroy = True
+        action5.is_format = True
+        action5.orig_format.type = "lvmpv"
+        action5.device.disk.name = "PHYSDISK-3"
+        action5.orig_format.vg_name = ""
+
+        storage = Mock()
+        storage.devicetree.actions = [action1, action2, action3, action4, action5]
+        error_handler = Mock()
+        warning_handler = Mock()
```

Comment:
I think that you might be able to use Blivet's devices and actions to model this state, but I don't insist on it.

Reply:
I'm open to suggestions. But would that be better? Mock lets me specify values arbitrarily deep in the "tree", and only these I care about, so this is still super short compared to alternatives...